addons/web/static/src/views/form/button_box/button_box.scss

```diff
@@ -137,16 +137,12 @@
             }
         }
 
-        .o_dropdown_more {
-            width: 150px;
-
-            .oe_stat_button {
-                height: auto;
-                width: 100%;
-                border: none;
-                padding-top: map-get($spacers, 2);
-                padding-bottom: map-get($spacers, 2);
-            }
+        .o_dropdown_more .oe_stat_button {
```

Comment:
Not blocking but keep the diff slim and readable is paramount when fixing production branches ;)
Despite removing the indentation makes perfectly sense here, it would have been preferred to just remove the `width: 150px;` line, without touching the other ones.